src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>();
```

Comment:
Consider initializing `people` with an empty array instead of `undefined`. This can simplify the logic when checking for the presence of people, as you won't need to use optional chaining (`?.`) to check the length.